pages/blog/index.vue

```diff
@@ -33,12 +34,51 @@ export default {
         'createdAt',
         'dir'
       ])
+      .where({
+        slug: 'index'
+      })
       .sortBy('createdAt', 'desc')
       .fetch()
       .catch(() => {
         error({ statusCode: 404, message: 'Page not found' })
       })
```

Comment:
An idea:
Always try to load the currentLang file, and load the defaultLang file in the catch ? So it prevent from loading unecessary english files that are translated.

Reply:
In terms of performance this would only really be affecting the build/generation time of the site and shouldn't affect the performance in production. Although I haven't tested that.

Will make the change anyway as it'll probably look cleaner overall.

Reply:
Didn't test, but with how the content module handles this particular case it won't actually catch an error, it just returns an empty array IIRC.